src/app/shared/services/app-config.service.ts

```diff
@@ -400,8 +400,13 @@ export class AppConfigService {
             getUrl: (record) => {
               let urls: Array<{ value: string }> = record['urls'];
               if (urls && urls.length > 0) {
-                return urls.map(url => url.value)
+                let url = urls.map(_url => _url.value)
                   .find(value => value.endsWith('.pdf'));
+                if (url !== undefined) {
+                  if (url.startsWith('http://')) {
```

Comment:
`replace` returns the same string if the first param doesn't match with any part, so you could removed the `startsWith` condition.